src/test/java/com/puppycrawl/tools/checkstyle/internal/CommitValidationTest.java

```diff
@@ -316,6 +319,7 @@ private static String getRulesForCommitMessageFormatting() {
                 + "    1) Must match one of the following patterns:\n"
                 + "        " + ISSUE_COMMIT_MESSAGE_REGEX_PATTERN + "\n"
                 + "        " + PR_COMMIT_MESSAGE_REGEX_PATTERN + "\n"
+                + "        " + RELEASE_COMMIT_MESSAGE_REGEX_PATTERN + "\n"
```

Comment:
I would not print this regexp to user.

Reply:
Done.

Reply:
There are Junit tests for this validation, please update them, all tests should be persistent if possible.

Reply:
Done. Added two test cases.